### DISCUSSION TITLE
cant connect in new server whit old tt5srv .xml

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2026] [Q&A]
hi hello i want creat new team talk server  and i created new file tt5srv 2.xml and copyed old xml file include user and password for old server. then run tt5srv-c tt5srv2.xml-wizard and sayDo you want to configure your TeamTalk Server? [Y/N] ("Y") 
Configuring file: tt5srv2.xml
and continiue to end but  when i want log in it get invalid account plese help me to fix it

**Comment by bear101** [comment]
Remember to use the -c option, i.e. ./tt5srv -c tt5srv2.xml
